actors/migration/nv3/migrators.go

```diff
@@ -0,0 +1,31 @@
+package nv3
```

Comment:
I get 3 for third in place, what does nv mean?

Reply:
"network version" (to distinguish from actors version)